v8go.cc

```diff
@@ -198,6 +198,28 @@ IsolateHStatistics IsolationGetHeapStatistics(IsolatePtr iso) {
                             hs.number_of_detached_contexts()};
 }
 
+/********** Exceptions & Errors **********/
+
+ValuePtr IsolateThrowException(IsolatePtr iso, ValuePtr value) {
+  if (iso == nullptr) {
+    return nullptr;
+  }
```

Comment:
We should do this check in the calling Go code, since it should panic to call methods like this on a disposed isolate.  I've pushed a commit to do this.

Reply:
Do you want to add a test for your fix using the helper recoverPanic function?